CONTRIBUTING.md

```diff
@@ -39,42 +39,62 @@ Next, with your virtual environment activated, install py42 and its development
 ["editable mode"](https://pip.pypa.io/en/stable/reference/pip_install/#editable-installs).
 
 ```bash
-$ pip install -e .[dev]
+pip install -e .[dev]
 ```
 
 Open the project in your IDE of choice and change the python environment to
 point to your virtual environment, and you should be ready to go!
 
-## General
+## Run a full build
+
+We use [tox](https://tox.readthedocs.io/en/latest/#) to run our build against Python 2.7, 3.5, 3.6, 3.7, and 3.8. When run locally, `tox` will run everything against the version of python that your virtual envrionment is running. When you open a PR, this process will be run against _all_ supported python versions.
```

Comment:
It might be worth separating out what happens locally versus what happens during a PR. We could have a section about PRs / the Review process which we will probably add to later

Reply:
On that note, also mentioning the branch name expectancies... even if we don't need a branch name style, it makes us seem more accepting to PRs. It could be like " We commend putting prefixing the PR branch name with chore, feature, or  bugfix."